src/__tests__/connection/passwordless/passwordless.actions.test.js

```diff
@@ -195,26 +196,42 @@ describe('passwordless actions', () => {
       expectMockToMatch(read, 1);
       expectMockToMatch(swap, 1);
     });
+
     it('calls webApi.passwordlessVerify() with sms options', () => {
       actions.logIn('id');
       expectMockToMatch(require('core/web_api').passwordlessVerify, 1);
     });
+
     it('calls webApi.passwordlessVerify() with email options', () => {
       require('connection/passwordless/index').isEmail = () => true;
       actions.logIn('id');
       expectMockToMatch(require('core/web_api').passwordlessVerify, 1);
     });
+
     describe('on webApi.passwordlessVerify() callback', () => {
-      it('formats error when there is an error ', () => {
-        actions.logIn('id');
+      describe('when there is an eror', () => {
```

Comment:
Typo. If you decide to change it, you'll need to regenerate the snapshots
```suggestion
      describe('when there is an error', () => {
```

Reply:
I've fixed the typo locally instead of merging the suggestion.

The event is already documented, it just wasn't being emitted in one of the places it should have been.